src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -563,4 +563,17 @@ class SentryPlugin implements Plugin<Project> {
 
         return names.findResult { project.tasks.findByName(it) }
     }
+
+    /**
+     * Returns the mapping file
+     * @param variant the ApplicationVariant
+     * @return the file or null if not found
+     */
+    static File getMappingFile(ApplicationVariant variant) {
+        try {
+            return variant.getMappingFileProvider().get().singleFile
```

Comment:
Grade new api of the hour?

Reply:
almost everything is a `Provider` now haha